dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataManager.kt

```diff
@@ -179,17 +186,20 @@ class DataManager(
      * Method to temporarily store a data set in a hash map and send a message to the storage_queue
      * @param dataId The id of the inserted data set
      * @param storableDataSet The data set to store
+     * @param bypassQa Whether the data set should be sent to QA or not
      * @param correlationId The correlation id of the request initiating the storing of data
      * @return ID of the stored data set
      */
```

Comment:
Wieso fangen die `MessageType` Optionen alle uppercase an und haben "gewöhnlich" geschriebene Werte, jedoch die `ExchangeNames` Optionen alle lowercase und haben Werte in camel case?

Wieso heißt es `MessageType` (singular) aber `ExchangeNames` (plural)

Reply:
Und aus Neugier,  nicht weil ich es schlecht inde:

Wieso sind `MessageType` und `ExchangeNames` Objekte, und keine Enum Klassen?

Ich habe diesen Fetzen im Internet gefunden:

"If the constants simply represent a set of related values without the need for additional functionality or type differentiation, using an enum might be more appropriate."

Persönlich finde ich beides ok.

Aber vllt gab es ja einen Gedanken dahinter?

Reply:
Irgendetwas sollte man hier auf jeden Fall ändern. Das waren früher auch enums. Ich hab da mit Marc schonmal diskutiert und aus irgendeinem Grund hielten wir dann objects für sinnvoller. Ich schau mal ob ich eine alte PR Diskussion ausgraben kann.

Reply:
Ok.   Die Umbennenung in meinem ersten Kommentar würde ich jedoch gut finden.    Ich schließe die conversation wenn das renaming vorgenommen wurde.

Reply:
Renamed. Ich glaube wir haben ein object statt eines enums verwendent, weil ein enum nicht direkt string Werte annehmen kann. Dann hätte statt `ExchangeName.QaCompleted` immer `ExchangeName.QaCompleted.name` stehen müssen.